backend/api/views/document.py

```diff
@@ -219,25 +135,18 @@ def create_new_documents():
     if "docClassIDs" not in data:
         return create_response(status=400, message="No document classes provided")
 
-    if "dueDate" not in data:
-        return create_response(status=400, message="No due date provided")
-
     userID = data.get("userID")
 
     status = "Missing"
 
-    date = data.get("dueDate")
-
     document_class_ids = data.get("docClassIDs").split(",")
     document_ids = []
 
+    if type(document_class_ids) != list:
```

Comment:
when a list is passed in through FormData, it is reduced to a string where individual elements are separated by commas
this checks for that